pkg/virt-launcher/virtwrap/nichotplug.go

```diff
@@ -95,6 +97,40 @@ func (vim *virtIOInterfaceManager) hotplugVirtioInterface(vmi *v1.VirtualMachine
 	return nil
 }
 
+func (vim *virtIOInterfaceManager) updateDomainLinkStates(currentDomain, desiredDomain *api.Domain) error {
+
+	currentDomainIfacesByAlias := indexedDomainInterfaces(currentDomain)
+	for _, desiredIface := range desiredDomain.Spec.Devices.Interfaces {
+		curIface, ok := currentDomainIfacesByAlias[desiredIface.Alias.GetName()]
+		if !ok {
```

Comment:
In a scenario where a simple hotplug is performed, the `currentDomain` is not expected to include the new interface, so we are suppose to fail here. But we are not suppose to fail here because the hotplug was already handled by `hotplugVirtioInterface`.